### PR TITLE
Blog, release announcements for 1.8.6 & 1.9.5, security bulletins

### PR DESCRIPTION
**Pull request by jacob-delgado**
* Add release notes for Istio-2021-003

* Update content/en/news/releases/1.9.x/announcing-1.9.3/index.md

Co-authored-by: Brian Avery <bavery@redhat.com>

* Update content/en/news/releases/1.8.x/announcing-1.8.5/index.md

Co-authored-by: Brian Avery <bavery@redhat.com>

Co-authored-by: Brian Avery <bavery@redhat.com>

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by brian-avery** [comment]
@googlebot I accept.

File: content/en/blog/2021/patch-tuesdays/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: How security releases (patch tuesday, embargoes, 0-days) are handled by the Product Security working group
```
Comment (ericvn) [inline content/en/blog/2021/patch-tuesdays/index.md:2]: Nit (can always push another PR later): Patch Tuesday (probably capitalize both). Not sure if Working Group would be capitalized as well.

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -71,6 +71,5 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases             | Patched versions with no known CVEs  |
 |----------------------------|--------------------------------------|
-| 1.9.x                      | 1.9.3+                               |
-| 1.8.x                      | 1.8.5+                               |
-| 1.7 and earlier            | None                                 |
+| 1.9.x                      | 1.9.5+                               |
+| 1.8 and earlier            | None                                 |
```
[the remark this answers is not in the document]
Comment (jacob-delgado) [reply content/en/docs/releases/supported-releases/index.md:75]: 1.8 is EOL (technically as of tomorrow), but 1.8.6 is the final release. The final release will fix known issues. The PSWG and T&R decided not to list it as it is no longer supported.

File: content/en/news/security/istio-security-2021-005/index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+the path `//admin` with multiple slashes should technically be treated as a different path from the `/admin`.
+However, some backend services choose to normalize the URL paths by merging multiple slashes to a single slash.
+This can result in a bypass of the authorization policy (`//admin` does not match `/admin`) and a user can access the resource at path `/admin`
+in the backend; this would represent a security incident.
+
+## Am I impacted?
+
+Your cluster is **impacted** by this vulnerability if you have authorization policies using `ALLOW action + notPaths field`
+or `DENY action + paths field` patterns.
+These patterns are vulnerable to unexpected policy bypasses and you should upgrade to fix the security issue ASAP.
```
[the remark this answers is not in the document]
Comment (jacob-delgado) [reply content/en/news/security/istio-security-2021-005/index.md:35]: done

File: content/en/news/releases/1.8.x/announcing-1.8.6/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+{{< relnote >}}
+
+{{< tip >}}
+This is the final release of 1.8. Please upgrade your Istio installation to a supported version.
+{{< /tip >}}
+
+## Security update
+
+The following 2 CVEs are highly related.
```
[the remark this answers is not in the document]
Comment (jacob-delgado) [reply content/en/news/releases/1.8.x/announcing-1.8.6/index.md:22]: Added a blank line

**Comment by ericvn** [comment]
I suspect this PR can be closed as the files have been cherry-picked back from the `release-1.9` branch with later updates.

**Pull request closed without merging**